### PR TITLE
Added per-file example

### PR DESCRIPTION
**Pull request by twolfson**
This is the second PR for #40 (complement to #50). In this PR:

- Added per-file example
- Explanation available in `README.rst`
    - https://github.com/twolfson/sops/tree/dev/per.file.example.sqwished/examples/per_file

### CODE REVIEW

File: examples/per_file/bin/decrypt-config.sh [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    rm -r config.bak
+  fi
+  mv config/ config.bak/
+fi
+
+# Create our new config directory
+mkdir config
+
+# For each of our files in our encrypted config
+for file in $(ls config.enc); do
```
Comment (twolfson) [inline examples/per_file/bin/decrypt-config.sh]: I recently found a better way to do this `for file in $(ls` that doesn't break when a file has a space in it. Updating the PR

**Comment by jvehent** [comment]
Hey @twolfson , thanks for another fantastic PR! I might take until next week to start reviewing, as I'm traveling this week, but it's definitely going to get a review soon.

**Comment by twolfson** [comment]
No worries, I don't think these 2 are high priority

File: examples/per_file/README.rst [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+    # From the `sops` root directory
+    # Import the test key
+    gpg --import tests/sops_functional_tests_key.asc
+
+    # Navigate to our example directory
+    cd examples/per_file
+
+    # Decrypt our secrets
+    bin/decrypt-secrets.sh
```
Comment (jvehent) [inline examples/per_file/README.rst]: `decrypt-config.sh`

**Comment by jvehent** [comment]
This looks fine as well, minus that one typo about `decrypt-config.sh`. 

We should add a reference to those examples in the main readme. Should I take care of it, or do you want to create a new PR for that?

File: examples/per_file/README.rst (hunk 2 above)
Comment (twolfson) [inline examples/per_file/README.rst]: Ah, nice catch

**Comment by twolfson** [comment]
I will take care of the typo by the end of the weekend (and post a comment when its done). I think it's best for you to take care of the main README; you know its organization better than myself

**Comment by twolfson** [comment]
We have addressed the comments and updated the PR

**Comment by jvehent** [comment]
Ha, looks like merging #50 made this one unhappy. Could you rebase it so I can merge it?

**Comment by twolfson** [comment]
Updated this PR to be on top of latest `master`

**Comment by jvehent** [comment]
Looks good, thanks again for the patch! :+1:

**Pull request merged**